From d6ba6dc554d0ee077f5ea692a8dfc4e0f59215ce Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Wed, 11 Sep 2019 20:14:27 +0200
Subject: [PATCH 136/152] renameFolderMenu: Don't save the source

The source actor is already tracked by the PopupMenu internally as
sourceActor, while nothing in RenameFolderMenu uses the source, so we can
drop this.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/720
---
 js/ui/appDisplay.js | 1 -
 1 file changed, 1 deletion(-)

diff --git a/js/ui/appDisplay.js b/js/ui/appDisplay.js
index 21469d1f9..39fa99e7e 100644
--- a/js/ui/appDisplay.js
+++ b/js/ui/appDisplay.js
@@ -1717,7 +1717,6 @@ var RenameFolderMenu = class RenameFolderMenu extends PopupMenu.PopupMenu {
     constructor(source, folder) {
         super(source.actor, 0.5, St.Side.BOTTOM);
 
-        this._source = source;
         this._folder = folder;
 
         // We want to keep the item hovered while the menu is up
-- 
2.23.0

